fe/fe-core/src/main/java/com/starrocks/persist/gson/GsonUtils.java

```diff
@@ -166,11 +169,12 @@ public class GsonUtils {
             .registerSubtype(StarOSTablet.class, StarOSTablet.class.getSimpleName());
 
     // runtime adapter for HeartbeatResponse
-    private static final RuntimeTypeAdapterFactory<HeartbeatResponse> heartbeatResponseAdapterFactor = RuntimeTypeAdapterFactory
-            .of(HeartbeatResponse.class, "clazz")
-            .registerSubtype(BackendHbResponse.class, BackendHbResponse.class.getSimpleName())
-            .registerSubtype(FrontendHbResponse.class, FrontendHbResponse.class.getSimpleName())
-            .registerSubtype(BrokerHbResponse.class, BrokerHbResponse.class.getSimpleName());
+    private static final RuntimeTypeAdapterFactory<HeartbeatResponse> heartbeatResponseAdapterFactor =
```

Comment:
why modify this?

Reply:
I formatted this file, I will restore this.